frontend/src/app/patients/Components/PersonForm.tsx

```diff
@@ -29,6 +29,36 @@ import ComboBox from "../../commonComponents/ComboBox";
 
 import FacilitySelect from "./FacilitySelect";
 
+const boolToYesNoUnknown = (
+  value: boolean | null | undefined
+): YesNoUnknown | undefined => {
+  if (value) {
+    return "YES";
+  }
+  if (value === false) {
+    return "NO";
+  }
+  if (value === null) {
+    return "UNKNOWN";
+  }
+  return undefined;
```

Comment:
Is the intention here that `UNKNOWN` is displayed only if the patient previously explicitly selected that; otherwise, an uninitialized state is used?

I'm not sure the backend can make that distinction since Java and Postgres only have the one emptiness sentinel value (`null`). Whether the value comes back over the wire as `null` or `undefined` would depend on a Jackson setting and not on what had been sent in previously.

Reply:
I would not expect to get anything but a boolean or `null` from the backend; this `return undefined` actually just handles the initialization when adding a new patient in the first place

Reply:
So, previously the db fields were `NOT NULL`, but you've removed that restriction, and null _is_ "UNKNOWN", and that's what we're initializing at startup?

I'm 👍 on this